src/compiler/compile/css/Selector.ts

```diff
@@ -124,6 +124,8 @@ export default class Selector {
 	}
 
 	validate(component: Component) {
+		this.blocks.some((block) => block.validateGlobalCompoundSelector(component));
```

Comment:
I would use `forEach` here and below since the return value of `some` is not used